h2o-r/h2o-package/R/frame.R

```diff
@@ -880,7 +879,7 @@ match.H2OFrame <- h2o.match
 #' @rdname h2o.match
 #' @export
 `%in%` <- function(x,table) {
-  if( is.H2OFrame(x) ) h2o.match(x,table,nomatch=0)
+  if( is.H2OFrame(x) ) h2o.match(x,table,nomatch=0) > 0
```

Comment:
did you verify that the following works?
```R
> hiris <- as.h2o(iris)
> 'setosa' %in% hiris$Species
```
should it be considered as a separate bug given that `%in%` uses `match` internally?
it should work the same way as
```R
> 'setosa' %in% iris$Species
[1] TRUE
```